pennylane/plugins/default_gaussian.py

```diff
@@ -841,7 +841,7 @@ def reduced_state(self, wires):
         r""" Returns the vector of means and the covariance matrix of the specified wires.
 
         Args:
-            wires (int of Sequence[int]): indices of the requested wires
+            wires (Iterable[int] or int): indices of the requested wires
```

Comment:
so `reduced_state(["mode5", "mode8"])` wouldn't be allowed?

Reply:
Good point. Need to go over default_gaussian again to adapt the docstrings.